examples/payout_usage.php

```diff
@@ -0,0 +1,108 @@
+<?
+
+use BTCPayServer\Client\PullPayment;
+
+class PullPayments {
+
+    public function getStorePullPayments()
+    {
+        $apiKey = '';
+        $host = '';
+        $storeId = '';
+        $includeArchived = TRUE;
+
+        try {
+            $client = new PullPayment($host, $apiKey);
+            var_dump($client->getStorePullPayments(
+                                $storeId, 
+                                $includeArchived));
+        } catch (\Throwable $e) {
+            echo "Error: " . $e->getMessage();
+        }
+    }
+    
+    public function createPullPayment()
+    {
+        $apiKey = '';
+        $host = '';
+        $storeId = '';
+        $paymentName = 'TestPayout-' . rand(0,10000000);
+        $paymentAmount = '0.000001';
```

Comment:
For amounts we have an utility class, you can use it like this: 
`$paymentAmount = PreciseNumber::parseString('0.000001');`